workflows/files/ZWECONF.properties

```diff
@@ -570,6 +619,14 @@ components_zss_crossMemoryServerName=ZWESIS_STD
 #  Check this to enable TLS on ZSS
 components_zss_tls=true
 
+# components_zss_agent_jwt_fallback
+# Label: ZSS JWT fallback
+# Abstract: Check this to enable fallback
+# Category: components
+# Description:
+#  Whether the agent will issue and accept cookies from itself in the event a JWT cannot be provided
```

Comment:
This is unclear .. If fallback is enabled, does the agent issue and accept cookies from itself in the event a JWT cannot be provided?